rust-runtime/aws-smithy-http-server/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "aws-smithy-http-server"
+version = "0.1.0"
+authors = ["Matteo Bigoi <matbigoi@amazon.com>", "David Pérez <davidpz@amazon.com>"]
+edition = "2018"
+description = """
+Server traits and utilities used by the code generator.
```

Comment:
can you also add some description about how this is extra unstable? And also maybe `publish = false` in the metadata?